Examples/EarlyStopping/heisenberg1d.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import netket as nk
+from netket.callbacks import EarlyStopping
+
+# TODO Help, nk.callbacks.EarlyStopping raises AttributeError: module 'netket' has no attribute 'callbacks'
+
+# 1D Lattice
+g = nk.graph.Hypercube(length=20, n_dim=1, pbc=True)
+
+# Hilbert space of spins on the graph
+# with total Sz equal to 0
+hi = nk.hilbert.Spin(s=0.5, graph=g, total_sz=0)
+
+# Heisenberg hamiltonian
+ha = nk.operator.Heisenberg(hilbert=hi)
+
+# Symmetric RBM Spin Machine
+ma = nk.machine.RbmSpin(alpha=1, hilbert=hi, symmetry=True)
+ma.init_random_parameters(seed=1234, sigma=0.01)
+
+# Metropolis Exchange Sampling
+# Notice that this sampler exchanges two neighboring sites
+# thus preservers the total magnetization
+sa = nk.sampler.MetropolisExchange(machine=ma)
+
+# Optimizer
+op = nk.optimizer.Sgd(ma, learning_rate=0.05)
+
+# Stochastic Reconfifugration
+sr = nk.optimizer.SR(ma, diag_shift=0.1)
+
+# Early Stopping
+es = EarlyStopping(patience=10)
+
+# Variational Monte Carlo
+gs = nk.Vmc(
+    hamiltonian=ha, sampler=sa, optimizer=op, n_samples=1000, n_discard=2, sr=sr,
+)
+
+gs.run(out="test", n_iter=300, callback=es.earlystopping)
```

Comment:
`gs.run(out="test", n_iter=300, callback=es)`

Reply:
assuming you change early stopping to `__call__`

Reply:
Also corrected this. However, I do not know how to create the callback object through `netket.callbacks.EarlyStopping()`. My terminal shows that `callbacks` is not a submodule of `netket`.

Reply:
you must add it to .__init__.py and to setup.py

Reply:
oops, I thought I had included it in `__init__.py`, but I did not!